spec/features/user_profiles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user can view profile' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view their profile' do
+    sign_in_as(user)
+    make_review_for_game(game)
```

Comment:
What is the benefit of this helper versus just doing something like 

```ruby
review = FactoryGirl.create(:review)
```

Also do you need to create a review to see a user profile? It shouldn't be the case that that is necessary.